rfcs/128-community-organization.md

```diff
@@ -0,0 +1,67 @@
+- Author: Xuanwo <github@xuanwo.io>
+- Start Date: 2021-06-29
+- RFC PR: [beyondstorage/specs#128](https://github.com/beyondstorage/specs/pull/128)
+- Tracking Issue: [beyondstorage/go-storage#0](https://github.com/beyondstorage/go-storage/issues/0)
+
+# GSP-128: Community Organization
+
+## Background
+
+BeyondStorage has been grown up, and we need to define our community organization to make it clear that everyone's rights and responsibilities.
+
+## Proposal
+
+I propose to add five positions for BeyondStorage, and they are separate in different projects.
+
+For now, there are two main projects:
+
+- `go-storage`: Focus on implement storage abstraction for golang, including `go-storage`, `go-service-*`, `go-endpoint`, `site`, `specs` and so on.
+- `dm`: Focus on implement data migration services, including `dm`.
+
+### Leader
+
+- ID: `leader`
+- Permissions: `Admin`
+
+Project core management team, involved in roadmap development of major community-related resolutions.
+
+### Maintainer
+
+- ID: `maintainer`
+- Permissions: `Maintain`
+
+The planner and designer of the project, with the authority to merge code into the trunk, from the Committer.
+
+### Committer
+
+- ID: `committer`
+- Permissions: `Write`
+
+The contributor who is recommended by Maintainer or Leader and has made an outstanding contribution to projects and needs to complete at least one feature or fix major bugs independently.
+
+### Reviewer
+
+- ID: `reviewer`
+- Permissions: `Triage`
+
+Advanced Contributor, responsible for reviewing community code, has LGTM (Looks Good To Me) approval for new submissions and can be involved in the design of subsequent projects.
```

Comment:
What does "can be involved in the design of subsequent projects" mean? I think everyone can propose a design  🤔

Reply:
They are copied from tikv/community, let me polish the details.